src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -106,7 +107,7 @@ public void WriteToReadOnlyFile()
             try
             {
                 // Operation succeeds when being run by the Unix superuser
-                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && geteuid() == 0)
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && PlatformDetection.IsSuperUser)
```

Comment:
I think it would be better to make `IsSuperUser` return false on Windows instead of throwing.

Reply:
Addressed in recent commits